oak_functions/client/rust/src/tests.rs

```diff
@@ -32,17 +32,26 @@ fn test_verify_rekor_log_entry() {
     // `bb05be1bd813f8afb7b77b2d9f7be5ae25b396d111c7a26a04b785c48c277372`.
     let log_entry_path = "testdata/logentry.json";
 
+    // Public key of the product team that signed the endorsement file.
+    let pubkey_path = "testdata/ec_public.pem";
+
     // Public key of the Rekor instance hosted by sigstore.dev. It is downloaded from https://rekor.sigstore.dev/api/v1/log/publicKey.
     let rekor_pubkey_path = "testdata/rekor_public_key.pem";
 
     let endorsement_bytes = fs::read(endorsement_path).expect("Couldn't read endorsement file.");
     let log_entry_bytes = fs::read(log_entry_path).expect("Couldn't read log entry file.");
     let rekor_pem_bytes =
         fs::read(rekor_pubkey_path).expect("Couldn't read Rekor's public key file.");
+    let pubkey_pem_bytes =
+        fs::read(pubkey_path).expect("Couldn't read product team's public key file.");
```

Comment:
Maybe "Couldn't read public key file."?

Reply:
Two public keys are used in this test scenario. I want to distinguish between them in the error message.